### ISSUE TITLE
Regression after Vim update 8.0.1704 -> 8.1.035: pressing Enter on a completion entry doesn't insert the text and the popup menu doesn't disappear

### ISSUE THREAD

**Issue by swilmet** [Rip-Rip/clang_complete#558]
I use Fedora 28. After a recent Vim update, `clang_complete` is unusable.

Vim version affected by the bug: 8.1.035
Vim version unaffected by the bug: 8.0.1704

I've tried re-installing `clang_complete`, but it doesn't solve the problem.

So when I press Enter on a completion entry, when the popup menu is shown, there are two problems:
1. It doesn't insert the text of the completion entry to the buffer/file (the symbol is not completed).
2. The popup menu doesn't disappear.

So basically `clang_complete` is unusable for me with Vim 8.1.035. I don't know if it's a bug in Vim, or if there was an API break and `clang_complete` needs to be adapted.

Do anybody else have the same bug? Or is it just me?

**Comment by swilmet** [comment]
Fixed with Vim 8.1.042.
